integration_tests/e2e/BDD/crm5.feature

```diff
@@ -0,0 +1,11 @@
+Feature: CRM5 Case Navigation
```

Comment:
Can we rename the BDD folder to crm5, so that the crm5.feature and crm5.ts file reside in `integration_tests/e2e/crm5`